"'argument': conversion from 'size_t' to 'long', possible loss of data" Windows build x64

My OS is Windows and I'm using the latest version (0.5.0) that vcpkg provides. There was no warning on the previous version (0.4.0) but this version has. If I build x86 then there is no problem; however, If I build x64 (both Debug and Release) then it gives warnings below and it's really annoying:

```
"blabla\vcpkg\installed\x64-windows\include\jwt-cpp\jwt.h(441,1): warning C4267: 'argument': conversion from 'size_t' to 'long', possible loss of data"
```

The problem is in the 'convert_base64_der_to_pem' function's line:

https://github.com/Thalhammer/jwt-cpp/blob/4620bf43befea5ef69816fb8610723232a609030/include/jwt-cpp/jwt.h#L441

--- a/include/jwt-cpp/jwt.h
+++ b/include/jwt-cpp/jwt.h
@@ -438,7 +438,7 @@ namespace jwt {
 			const auto decodedStr = decode(cert_base64_der_str);
 			auto c_str = reinterpret_cast<const unsigned char*>(decodedStr.c_str());
 
-			std::unique_ptr<X509, decltype(&X509_free)> cert(d2i_X509(NULL, &c_str, decodedStr.size()), X509_free);
+			std::unique_ptr<X509, decltype(&X509_free)> cert(d2i_X509(NULL, &c_str, static_cast<int>(decodedStr.size())), X509_free);
 			std::unique_ptr<BIO, decltype(&BIO_free_all)> certbio(BIO_new(BIO_s_mem()), BIO_free_all);
 			if (!cert || !certbio) {
 				ec = error::rsa_error::create_mem_bio_failed;

--- a/include/jwt-cpp/jwt.h
+++ b/include/jwt-cpp/jwt.h
@@ -438,7 +438,8 @@ namespace jwt {
 			const auto decodedStr = decode(cert_base64_der_str);
 			auto c_str = reinterpret_cast<const unsigned char*>(decodedStr.c_str());
 
-			std::unique_ptr<X509, decltype(&X509_free)> cert(d2i_X509(NULL, &c_str, static_cast<int>(decodedStr.size())), X509_free);
+			std::unique_ptr<X509, decltype(&X509_free)> cert(
+				d2i_X509(NULL, &c_str, static_cast<int>(decodedStr.size())), X509_free);
 			std::unique_ptr<BIO, decltype(&BIO_free_all)> certbio(BIO_new(BIO_s_mem()), BIO_free_all);
 			if (!cert || !certbio) {
 				ec = error::rsa_error::create_mem_bio_failed;

